src/universal_transfer_operator/data_providers/database/base.py

```diff
@@ -743,5 +746,21 @@ def export_table_to_pandas_dataframe(self) -> pd.DataFrame:
             raise ValueError(f"The table {self.dataset.name} does not exist")
 
         sqla_table = self.get_sqla_table(self.dataset)
-        df = pd.read_sql(sql=sqla_table.select(), con=self.sqlalchemy_engine)
-        return PandasDataframe.from_pandas_df(df)
+        # df = pd.read_sql(sql=sqla_table.select(), con=self.sqlalchemy_engine)
+        # from universal_transfer_operator.data_providers.dataframe.Pandasdataframe import (
+        #     PandasdataframeDataProvider,
+        # )
+        #
+        # return PandasdataframeDataProvider.from_pandas_df(df)
+        return pd.read_sql(sql=sqla_table.select(), con=self.sqlalchemy_engine)
+
+    def is_native_path_available(
+        self,
+        source_dataset: Table,  # skipcq PYL-W0613, PYL-R0201
+    ) -> bool:
+        """
+        Check if there is an optimised path for source to destination.
+
+        :param source_dataset: Dataframe from which we need to transfer data
+        """
+        return False
```

Comment:
Why do we always return False?

Reply:
By default, these data providers don't have native transfers. If any specific data provider has native transfer enabled, it overrides this.